services/user.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: int,
```

Comment:
The `password` parameter should be of type `str` instead of `int`. Passwords are typically strings, and using an integer type is incorrect.